core/file/ctime_spec.rb

```diff
@@ -14,6 +14,21 @@
     File.ctime(@file).should be_kind_of(Time)
   end
 
+  platform_is :linux do
+    it "Returns the change time for the named file (the time at which directory information about the file was changed, not the file itself) with microseconds." do
```

Comment:
Copy-pasted from previous `it` :(

What can I do with some fixes if that PR was merged? New PR?